releasenotes/notes/add-suffix-averaging-ba6a7027ea11d5db.yaml

```diff
@@ -0,0 +1,32 @@
+---
+features:
```

Comment:
I think maybe we should also include the fact that the optimizers have been updated to take callback explicitly (you could do it with kwargs before) and the callback can  be read/updated.